plugins/CoreHome/javascripts/broadcast.js

```diff
@@ -467,7 +512,11 @@ var broadcast = {
             }
         }
 
-        $location.search('popover', popover);
+        var $window = piwikHelper.getAngularDependency('$window');
```

Comment:
`$location.search()` as a setter breaks array parameters completely. Even when you're just setting a non-array parameter, if there are array params in current query, they'll be removed (or set as `myArrayParam=` instead of `myArrayParam[]=`. To use array query params I had to avoid using `$location.search(name, value)`. Replacing the whole string works though.